### PR TITLE
Cherry-picks.md: Moved to k/community/sig-release and stub file created

### PR DESCRIPTION
**Pull request by eduartua**
Ref. #3064 
This is a more process-related file rather than technical.

/cc @parispittman @nikhita 
/sig contributor-experience

/assign @cblecker @justaugustus 

/area developer-guide
/kind cleanup

### CODE REVIEW

**Comment by eduartua** [comment]
I am updating all files with links to this file.

File: contributors/devel/cherry-picks.md [hunk 1]
```diff
@@ -1,73 +1,3 @@
...
-
-Contributors may encounter some of the following difficulties when initiating a cherry-pick.
-
-- A cherry-pick PR does not apply cleanly against an old release branch.
-In that case, you will need to manually fix conflicts.
-
-- The cherry-pick PR includes code that does not pass CI tests.
-In such a case you will have to fetch the auto-generated branch from your fork, amend the problematic commit and force push to the auto-generated branch.
-Alternatively, you can create a new PR, which is noisier.
+*This file is a redirect stub. It should be deleted within 90 days from the current date.*
```
Comment (nikhita) [inline contributors/devel/cherry-picks.md]: Can you add a date here instead i.e. this file should be deleted by <...>

File: sig-release/cherry-picks.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# Overview
```
Comment (nikhita) [inline sig-release/cherry-picks.md]: My understanding was that these files will move from k/community/devel to k/community/devel/sig-release/cherry-picks.md. :thinking:

File: contributors/devel/cherry-picks.md (hunk 1 above)
Comment (eduartua) [reply contributors/devel/cherry-picks.md]: Sure

File: sig-release/cherry-picks.md (hunk 2 above)
Comment (eduartua) [reply sig-release/cherry-picks.md]: So, it is better to create folders in accordance with the [classification](https://github.com/kubernetes/community/issues/3097) of files by SIGs, right?

Comment (nikhita) [reply sig-release/cherry-picks.md]: Yes, just like how design proposals are grouped by SIG: https://github.com/kubernetes/community/tree/master/contributors/design-proposals

**Pull request closed without merging**